c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -1,7 +1,20 @@
-VENV_BIN ?= .build/venv/bin
 PYTHON_VERSION = $(shell $(VENV_BIN)/python -c "import sys; print('%i.%i' % (sys.version_info.major, sys.version_info.minor))" 2> /dev/null)
 PACKAGE = {{package}}
+OPERATING_SYSTEM ?= LINUX
+
+ifeq ($(OPERATING_SYSTEM), WINDOWS)
+VENV_BIN = .build/venv/Scripts
+PYTHON_BIN_POSTFIX = .exe
+NODE_BIN_POSTFIX = .cmd
+FIND ?=$(CYGWIN_PATH)/bin/find.exe
```

Comment:
@ochriste, are you going to do something about the documentation ? I saw that it should undergo a major rework for Windows...

I wanted to do it, but I am running around and do not think that I will have time to do it before April...